gradle/dependencies.gradle

```diff
@@ -80,11 +82,17 @@ versions += [
   zkclient: "0.10",
   zookeeper: "3.4.10",
   jfreechart: "1.0.0",
-  mavenArtifact: "3.5.3"
+  mavenArtifact: "3.5.3",
+  avro4sVersion: "1.8.3",
+  scalatestEmbeddedKafkaVersion: "1.1.0"
 ]
 
 libs += [
   activation: "javax.activation:activation:$versions.activation",
+  jfreechart: "1.0.0"
+]
+
+libs += [
```

Comment:
Can't we merge this with the one from above?

Reply:
+1